package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "one-dark-syntax",
   "theme": "syntax",
-  "version": "1.7.1",
+  "version": "1.8.1",
```

Comment:
Updating the version doesn't need to be part of this PR. It will be done at a later point.